### PR TITLE
add version fix

### PR DESCRIPTION
**Pull request by joequant**
This fixes a typo in the version code.  Also the config file has
a space which causes the minor version to fail.  Rather than
modify that file, this makes the minor version check more robust.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -29,12 +29,12 @@ set(CMAKE_PROGRAM_PATH ${CMAKE_PREFIX_PATH})
 file(STRINGS "${XEUS_SQLITE_INCLUDE_DIR}/xeus-sqlite/xeus_sqlite_config.hpp" xsqlite_version_defines
      REGEX "#define XSQLITE_VERSION_(MAJOR|MINOR|PATCH)")
 foreach(ver ${xsqlite_version_defines})
-    if(ver MATCHES "#define XSQLITE_VERSION_(MAJOR|MINOR|PATCH) +([^ ]+)$")
```
Comment (marimeireles) [inline CMakeLists.txt:32]: Why did you remove the `$`?
We're interested in that one, no?

**Comment by joequant** [comment]
Oh.

the problem is that the minor version is

#define foo 1(space)

Which means it didn't match the regex

Rather than remove the trailing space, I changed the regex to be more
robust.

**Comment by marimeireles** [comment]
Alright, I see your point.
The problem now is that... It catches all other words if there are any. e.g is matching `#define XSQLITE_VERSION_MAJOR 1.2.3 banana` and we don't want that.
What do you think of adding: `#define XSQLITE_VERSION_(MAJOR|MINOR|PATCH) +([^ ]+) *$`? It's a bit more error proof, imo.
Thanks again!

**Comment by joequant** [comment]
That seems reasonable to me.

**Comment by marimeireles** [comment]
Thanks again @joequant ! 🌸
